src/Model.php

```diff
@@ -774,22 +803,15 @@ public function get(string $field = null)
         if ($field === null) {
             // Collect list of eligible fields
             $data = [];
-            if ($this->only_fields) {
-                // collect data for actual fields
-                foreach ($this->only_fields as $field) {
-                    $data[$field] = $this->get($field);
-                }
-            } else {
-                // get all fields
-                foreach ($this->getFields() as $field => $f) {
-                    $data[$field] = $this->get($field);
-                }
+            foreach ($this->getFields(self::FIELD_FILTER_ONLY_FIELDS) as $field) {
```

Comment:
`FIELD_FILTER_ONLY_FIELDS` should be removed, and `isOnlyFieldsField` should be used directly